### PR TITLE
Update use-windows-event-forwarding-to-assist-in-intrusion-detection.md

### PR DESCRIPTION
**Pull request by warren-msft**
Updated the "What are the WEC server's limitations section per Gianni Bragante's recommendation.

### CODE REVIEW

**Review by illfated — APPROVED** [review]
Looks good to me.

File: windows/security/threat-protection/use-windows-event-forwarding-to-assist-in-intrusion-detection.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Yes. If you desire a High-Availability environment, simply configure multiple WE
 
 ### <a href="" id="what-are-the-wec-server-s-limitations-"></a>What are the WEC server’s limitations?
 
-There are three factors that limit the scalability of WEC servers. The general rule for a stable WEC server on commodity hardware is “10k x 10k” – meaning, no more than 10,000 concurrently active WEF Clients per WEC server and no more than 10,000 events/second average event volume.
+There are three factors that limit the scalability of WEC servers. The general rule for a stable WEC server on commodity hardware is planning for a total of 3000 events per second for all configured subscriptions on average
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/use-windows-event-forwarding-to-assist-in-intrusion-detection.md:150]:
```suggestion
There are three factors that limit the scalability of WEC servers. The general rule for a stable WEC server on commodity hardware is planning for a total of 3,000 events per second on average for all configured subscriptions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by ianni1998g — COMMENTED** [review]
Bien

**Pull request merged**